plugin/clone/include/clone_client.h

```diff
@@ -51,6 +51,10 @@ using Time_Msec = std::chrono::milliseconds;
 using Time_Sec = std::chrono::seconds;
 using Time_Min = std::chrono::minutes;
 
+static const char *clone_compression_algorithms[] = {
```

Comment:
If it's only used in a single source file, it should be declared there instead of a header. If it is used in multiple source files, it should not be `static`